src/main/resources/templates/journals/create.html

```diff
@@ -0,0 +1,24 @@
+<!DOCTYPE html>
+<html lang="en" xmlns:th="http://www.thymeleaf.org" xmlns:sec="http://www.thymeleaf.org/extras/spring-security">
+<head  th:replace="partials/elements :: head(title='FilmSage Index')">
+    <meta charset="UTF-8"/>
+    <title>Create a Journal</title>
```

Comment:
Delete this line. 
change head(title='FilmSage Index') to head(title='Create a Journal')